client/src/components/ui/navigationbar/logoutIcon.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+    import type { User } from '../../../../../model/src/user.ts';
+
+    // eslint-disable-next-line no-undefined
+    export let picture = undefined as User['picture'] | undefined;
+    let isLogoutHovered = false as boolean;
+    const logoutIcon = new URL('../../../assets/icons/carbon/logout-profile.svg', import.meta.url);

```

Comment:
Again, we should have a dedicated component for this in `components/icons/` instead of importing it directly as an `<img>`.